File: Service/ClaimCare/RMA.Service.ClaimCare.Contracts/Entities/AdhocClaimRequirementCommunicationRequest.cs
using RMA.Service.ClientCare.Contracts.Entities.RolePlayer;

using System.Collections.Generic;

namespace RMA.Service.ClaimCare.Contracts.Entities
{
    public class AdhocClaimRequirementCommunicationRequest
    {
        public string Subject { get; set; }
        public List<RolePlayerContact> RolePlayerContacts { get; set; }
        public string RequirementsHtml { get; set; }
        public string RequirementsCsv { get; set; }
        public int PersonEventId { get; set; }
        public string DisplayName { get; set; }
    }
}

File: Web/RMA-Web-Shell/projects/claimcare/src/app/claim-manager/shared/entities/adhoc-claim-requirement-communication-request.ts
import { RolePlayerContact } from 'projects/clientcare/src/app/member-manager/models/roleplayer-contact';

export class AdhocClaimRequirementCommunicationRequest {
  rolePlayerContacts: RolePlayerContact[];
  requirementsHtml: string;
  requirementsCsv: string;
  personEventId: number;
  subject: string;
  displayName: string;
}
